src/System.Private.CoreLib/shared/Interop/Unix/System.Globalization.Native/Interop.Locale.cs

```diff
@@ -4,37 +4,36 @@
 
 using System;
 using System.Runtime.InteropServices;
-using System.Text;
 
 internal static partial class Interop
 {
     internal static partial class Globalization
     {
         [DllImport(Libraries.GlobalizationNative, CharSet = CharSet.Unicode, EntryPoint = "GlobalizationNative_GetLocaleName")]
         [return: MarshalAs(UnmanagedType.Bool)]
-        internal static extern unsafe bool GetLocaleName(string localeName, [Out] StringBuilder value, int valueLength);
+        internal static extern unsafe bool GetLocaleName(string localeName, char* value, int valueLength);
```

Comment:
(nit) I kind of like the `[Out]` attributes as a way of documentation.  But I'm totally fine with removing them.